app.py

```diff
@@ -315,8 +316,23 @@ def post(year, month, day, slug):
 
 @app.route('/author/<slug>/')
 def user(slug):
-    api_url = ''.join([INSIGHTS_URL, '/users?_embed&slug=', slug])
+    api_url = ''.join([API_URL, '/users?_embed&slug=', slug])
     response = _get_from_cache(api_url)
     data = json.loads(response.text)[0]
     data = _normalise_user(data)
     return flask.render_template('author.html', author=data)
+
+
+@app.route('/wp-'
```

Comment:
This seems like an unnecessary use of regex, when it's just to match two routes. I think it would be clearer to make them into separate routes - one for `/wp-content/<path:path>` and one for `wp-admin/<path:path>`. You may also want to make use of `defaults={'path': ''}` (see http://flask.pocoo.org/snippets/57/).

In our other projects, we also use a `redirects.yaml` file to store redirects - e.g. how I've done it here: https://github.com/canonical-docs/phone-docs/pull/11/files#diff-3f41e546893dc64b71aaacad12cad815R33. You can decide if making use of that for `wp-admin` etc. is appropriate. But down the line we will probably want to implement some sort of `redirects.yaml` anyway, as a simple way to add redirects, like how we do in www.ubuntu.com

Reply:
Now we don't use regex anymore to make it less confusing.
I created a issue to package your redirection code: https://github.com/ubuntudesign/web-squad/issues/213